NYPL/fedora_ingest_rails
# DR 2830 implement delayed job web

## Jira Tickets ##
- [Expose fedora ingest delayed jobs through front end](https://jira.nypl.org/browse/DR-2830)

## Things Done ##
- Added delayed job web
- Added delayed job web GET routes
- Made delayed job web POST routes redirect to /delayed_job

## How to Test ##
- Build a new webapp container
- Bring up the app but turn off the worker
- Add one or more delayed jobs to the database (for example, posting a uuid to /ingest_requests)
- Navigate to /delayed_job. You should be able to see your job(s) enqueued
- Try to retry, remove, or reload a job. You should be redirected to /delayed_job without any effect.
